Capstone_EDA.ipynb

```diff
@@ -520,8 +520,15 @@
    "metadata": {},
```

Comment:
<p>Looks fine to me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/davidslabon/Capstone_UnlockingClimateSolutions/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>